template.yaml

```diff
@@ -339,6 +351,27 @@ Resources:
           CidrIp: 0.0.0.0/0
           IpProtocol: '-1'
 
+  SecurityGroupLambdaBackup:
+    Type: AWS::EC2::SecurityGroup
+    Condition: EnableBackup
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W5
+            reason: All protocols desired for outbound traffic.
+          - id: W40
+            reason: All protocols desired for outbound traffic.
+    Properties:
+      GroupDescription: Lambda Security Group to be referenced in EFS Sec Group
+      VpcId: !If
+        - CreateVpc
+        - !GetAtt VpcStack.Outputs.VpcId
+        - !Ref Vpc
+      SecurityGroupEgress:
+        - Description: All outbound
+          CidrIp: 0.0.0.0/0
+          IpProtocol: '-1'
```

Comment:
I wonder if it might be a good idea to limit this Lambda' outbound options? Perhaps limited to port 443 to anywhere (S3 write) and 2049 to EFS SG only?